tests/sweep_framework/sweeps/eltwise/unary/lgamma/lgamma_sharded.py

```diff
@@ -29,7 +29,7 @@
 # Developers can create their own generator functions and pass them to the parameters as inputs.
 parameters = {
     "nightly": {
-        "input_spec": gen_sharded_spec_unary(16, max_tensor_size=2 * 1024 * 1024, layouts=["TILE_LAYOUT"]),
+        "input_spec": gen_sharded_spec_unary(16, max_tensor_size_per_core=20 * 1024, layouts=["TILE_LAYOUT"]),
```

Comment:
could you please explain where 20 (and 14 in file below) comes from?

Reply:
It's heuristic. Tests run too slow with larger tensors sizes... Since this ops are slower...